lib/libzrepl/data_conn.c

```diff
@@ -671,7 +698,7 @@ uzfs_zvol_timer_thread(void)
 					    zinfo->name);
 					uzfs_zvol_store_last_committed_io_no(
 					    zinfo->zv,
-					    DEGRADED_IO_SEQNUM,
```

Comment:
along with change in key name, we need to change the variable names also. Lets discuss on this once again.